assets/js/edit.js

```diff
@@ -273,6 +302,23 @@ class Edit extends Component {
 								{__('Add Transcript', 'simple-podcasting')}
 							</Button>
 						</PanelRow>
+						<h3 style={{marginTop: '20px'}}>{__('Cover Image', 'simple-podcasting')}</h3>
+						<p>{__('The featured image of the current post is used as the episode cover art. Please select a featured image to set it.', 'simple-podcasting')}</p>
+						<PanelRow>
+							{featuredImageUrl && (
+								<img src={featuredImageUrl} alt="Cover Image" />
+							)}
+
+							<MediaUploadCheck>
+								<MediaUpload
+									onSelect={onUpdateImage}
+									allowedTypes={['image/gif']}
+									render={({ open }) => (
+										<Button onClick={open}>{__('Select Cover Art', 'simple-podcasting')}</Button>
```

Comment:
Maybe some styling is needed here? as the button gets hidden inside when we have an image available.
<img width="321" alt="Screenshot 2024-02-26 at 10 44 01 PM" src="https://github.com/10up/simple-podcasting/assets/10613171/5436c7fc-d3f0-43f1-a503-fd8c70b286b6">

Reply:
@iamdharmesh Added the style and while on it I introduced the Delete button for the image and updated button texts based on whether we have an image or want to add a new one as a convenience.
cc: @jeffpaul 

<img width="279" alt="Screenshot 2024-02-27 at 1 35 01 PM" src="https://github.com/10up/simple-podcasting/assets/45451844/e51ff623-57bd-4162-a325-424b85fa5280">